.ci/Dockerfile

```diff
@@ -1,7 +1,7 @@
 # NOTE: This Dockerfile is ONLY used to run certain tasks in CI. It is not used to run Kibana or as a distributable.
 # If you're looking for the Kibana Docker image distributable, please see: src/dev/build/tasks/os_packages/docker_generator/templates/dockerfile.template.ts
 
-ARG NODE_VERSION=14.17.3
+ARG NODE_VERSION=14.17.5
```

Comment:
__note:__ Docker Hub isn't updated yet, I'll revert this line if 14.17.5 isn't available there by the time we're ready to merge.

Reply:
Done